internal/model/loadbalancer.go

```diff
@@ -20,12 +20,38 @@ import (
 	"github.com/golang-jwt/jwt"
 )
 
+type AllowedIPBlock struct {
+	Cidr   string   `json:"cidr"`
+	Except []string `json:"except"`
+}
+
+type PolicyPort struct {
+	Protocol corev1.Protocol `json:"protocol"`
+	Port     *int32          `json:"port,omitempty"`
+	EndPort  *int32          `json:"end-port,omitempty"`
+}
+
+type IngressRule struct {
+	AllowedIPBlocks []AllowedIPBlock `json:"allowed-ip-blocks"`
```

Comment:
If you disagree with moving the "no from => all addresses allowed" logic to the controller, please add a comment here which explains that an empty AllowedIPBlocks list implies "allow everything", which is not obvious from the name.

Reply:
Added comments as shown in https://github.com/cloudandheat/ch-k8s-lbaas/pull/36#discussion_r999469231